website/src/components/Chat/ChatConfigDrawer.tsx

```diff
@@ -118,7 +118,9 @@ const ChatConfigForm = () => {
   const { control, register, reset, getValues } = useFormContext<ChatConfigFormData>();
   const selectedModel = useWatch({ name: "model_config_name", control: control });
   const presets = modelInfos.find((model) => model.name === selectedModel)!.parameter_configs;
-
+  const [selectedPresetName, setSelectedPresetName] = useState(
+    () => presets.find((preset) => areParametersEqual(preset.sampling_parameters, getValues()))?.name ?? presets[0].name
```

Comment:
should `presets[0].name` be `customPresetName`?

Reply:
Done